packages/contracts/contracts/DefaultFeeOracle.sol

```diff
@@ -7,13 +7,24 @@ import "./Ownable.sol";
 
 contract DefaultFeeOracle is Ownable, IFeeOracle {
```

Comment:
Maybe we could change the name to DefaultDUOracle or DefaultOracle and change the interface name too since it's not only handling the fee anymore.

Reply:
It **is** only handling the fee though:
- How much is the fee
- Where should the fee be paid

Both things definitely relate to the fee!